README.md

```diff
@@ -11,129 +11,64 @@ interface for KernelCI, to replace
 mainly based on [FastAPI](https://fastapi.tiangolo.com/), [Mongo
 DB](https://www.mongodb.com/) and [Redis](https://redis.io/).
 
+Please refer to the architecture here: https://kernelci.org/docs/api/overview/#api-architecture.
 
-## Getting started with docker-compose
-
-The FastAPI server and Mongo DB can all be started at the same time using
-`docker-compose`.  Then FastAPI can access the Mongo DB container directly via
-the `db` hostname.
-
-```
-$ docker-compose up --build
-[...]
-kernelci-api | INFO:     Application startup complete.
-```
 
-Then to check it's working:
+## Getting started with docker-compose
 
-```
-$ curl http://localhost:8001/
-{"message":"KernelCI API"}
-```
+Please refer https://kernelci.org/docs/api/getting-started/#start-docker-compose to start Docker containers of all the services.
 
 Note that the FastAPI server is running on port 8000 inside the container, but
 it's exposed to the host via port 8001 to avoid conflicts with other services.
 This can be adjusted for each setup in
 [`docker-compose.yaml`](docker-compose.yaml).
 
-The container log should show:
-```
-kernelci-api | INFO:     172.20.0.1:49228 - "GET / HTTP/1.1" 200 OK
-```
-
-## API Documentation
-
-The FastAPI server will automatically generate documentation for the API itself
-and serve it directly:: http://localhost:8001/docs
-
-This is based on OpenAPI, and you can also download the `openapi.json` file to
-use it with other tools: http://localhost:8001/openapi.json
-
 
 ## Authentication
 
 ### Generate SECRET KEY and add it in environment file
```

Comment:
This is also covered by the Getting Started documentation so I think it can be dropped.

Reply:
I haven't dropped the section but added a link and description about env.sample (which is not covered in the documentation).